types/lodash/index.d.ts

```diff
@@ -18471,7 +18471,7 @@ declare namespace _ {
          * @param defaultValue The default value.
          * @returns Returns the resolved value.
          */
-        defaultTo<T>(value: T, defaultValue: T): T;
+        defaultTo<T>(value: T | null | undefined, defaultValue: T): T;
```

Comment:
The tests for this started failing once I enabled `strictNullChecks`, so I fixed it to make the tests work.

On a side note, the tests are still failing for me with a different error now. But the error seems unrelated to my changes, and I also get the same error on DefinitelyTyped:master (which excludes my changes). The error is: Error: ENOTDIR: not a directory, scandir '/home/travis/build/aj-r/DefinitelyTyped/types/tslint.json'.